cards/angioedema/card.md

```diff
@@ -5,6 +5,7 @@ authors:
     - 'Michelle Lin, MD'
 created: 2010/03/26
 updates: null
++updates: 2017/2/24
```

Comment:
This needs to be fixed.

@andyglittle It looks like you're copying the diff line that includes the `+` to show that the change as been added. Try to avoid that. The `+` should not be there.

Also, `updates` is an array of strings, not a string.

Also, I believe we're keeping categories lowercase, right @jvoros?

It should look like this:

```yml
title: Angioedema
authors:
    - 'Michelle Lin, MD'
created: 2010/03/26
updates:
    - 2017/2/24
categories:
    - allergy-immunology
```